sklearn/model_selection/_split.py

```diff
@@ -2013,7 +2013,7 @@ class GroupShuffleSplit(GroupsConsumerMixin, BaseShuffleSplit):
     ``GroupShuffleSplit(test_size=10, n_splits=100)``.
 
     Note: The parameters ``test_size`` and ``train_size`` refer to groups, and
-    not to samples, as in ShuffleSplit.
+    not to samples, as in :class:`ShuffleSplit`.
```

Comment:
I think the current phrasing can be ambiguous, as it is unclear where does the parenthetical commas close. One possibility is to remove one of them, as per the suggestion below:

```suggestion
    not to samples as in :class:`ShuffleSplit`.
```

Other possibilies could be to explicitly add parenthesis:
> Note: The parameters ``test_size`` and ``train_size`` in `GroupShuffleSplit` refer to groups (and not to samples, as in `ShuffleSplit`).

Or changing the whole phrasing:
> Contrary to `ShuffleSplit`, where the parameters ``test_size`` and ``train_size`` refer to proportion of samples, those same parameter in `GroupShuffleSplit` refer to group proportions.

Reply:
Agree, that is confusing. I've amended to your first suggestion, thanks

Reply:
Thanks. The PR is looking good :) Don't forget to also address my comment on changing the docstring description of `test_size`.

Reply:
Whoops, must have been half asleep. Fixed!